setup.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+
+# Installing poetry
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+# Add poetry to PATH in script
+export POETRY_ROOT=\"$HOME//.poetry/\"
+export PATH=\"$POETRY_ROOT/bin:$PATH\"
+
+#Checking if poetry is already installed
+poetry --version 
+
+#Install Python
+sudo apt-get update -y && sudo apt-get install -y python3.8 
```

Comment:
It's worth noting, though not an issue, that this ties the script to systems that have the `apt` package manager (e.g. Debian/Ubuntu)